scripts/add_confirmed_to_existing_users_token.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Migration to add a confirmed boolean to User.email_verifications
+"""
+import sys
+import logging
+from modularodm import Q
+
+from website.app import init_app
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from framework.auth import User
+
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration():
+    for user in User.find(Q('email_verifications', 'ne', {})):
+        for token in user.email_verifications:
+            if 'confirmed' not in token:
+                user.email_verifications[token]['confirmed'] = False
+
+
+def main(dry=True):
+    init_app(set_backends=True, routes=False)  # Sets the storage backends on all models
```

Comment:
`init_app` is usually called before a `TokuTransaction` is opened, but it's *probably* fine?